### ISSUE TITLE
Unreadable error messages on certain terminal emulators

### ISSUE THREAD

**Issue by koute** [paritytech/cargo-unleash#61]
I was quite confused getting the following error messages from `cargo-unleash` when trying to run it locally:

![unleash](https://user-images.githubusercontent.com/246574/145569073-01716561-4944-416c-8943-fdb4a484d152.png)

Apparently those error messages were printed out with a black text color, and since I my terminal background is also black the text is effectively invisible. If I copy-paste the text somewhere else (or redirect `cargo-unleash`'s output to a file) I can read the error messages just fine:

```
ERROR [cargo_unleash::commands::check] "node-runtime: Bad metadata: description is missing"
ERROR [cargo_unleash::commands::check] "node-bench: Bad metadata: repository is missing"
ERROR [cargo_unleash::commands::check] "node-inspect: Bad metadata: description is missing"
ERROR [cargo_unleash::commands::check] "node-rpc: Bad metadata: description is missing"
Error: "Soft checkes failed with 4 errors (see above)"
```

**Comment by gnunicorn** [comment]
hmmm... that's odd. Quick investigation tells me, this is just the default print-format for the `anyhow:Error`, we just bubble all the way up to [`main`](https://github.com/paritytech/cargo-unleash/blob/master/src/main.rs#L8), just containing a bunch of string - we aren't doing anything else to it on the cargo-unleash-side :thinking:  ...

**Comment by koute** [comment]
Yeah, I found it quite strange too; it's *probably* not an issue on `cargo-unleash` side per-se, I just wanted to document it here in case anyone also gets confused why the errors are not being displayed.
